jujubigdata/utils.py

```diff
@@ -187,7 +189,7 @@ def re_edit_in_place(filename, subs, encoding='utf8'):
     """
     with Path(filename).in_place(encoding=encoding) as (reader, writer):
         for line in reader:
-            for pat, repl in subs.iteritems():
+            for pat, repl in six.iteritems(subs):
```

Comment:
I'd prefer to just switch to `subs.items()`.  I don't think we really have to worry about `subs` ever being big enough to cause issues from being duplicated.